src/styling/songResultList.css

```diff
@@ -0,0 +1,22 @@
+.results-container {
+  margin: 10px 10px;
+}
+
+.results-list {
+  max-height: 500px;
+  overflow: hidden;
+  overflow-y: scroll;
+  margin: 10px 10px;
+}
+
+.results-list-item-text-name {
+  margin: 0px 10px;
+  min-width: 35% !important;
+  max-width: 35% !important;
+  word-wrap: break-word !important;
+  overflow: hidden;
+}
+
+.results-list-item-text-album {
+  margin: 0px 20px;
+}
```

Comment:
Instead of using raw CSS here and overriding things with `!important`, can we use the `sx ` prop for the relevant MUI component? https://mui.com/system/getting-started/the-sx-prop/ 

You can specify `minWidth`, etc in there

Reply:
Are we not able to put word wrap, overflow, the other margin CSS in the sx props and get rid of the entire css file?

Reply:
hmm having some issues implementing it, must be some conflicting styles. How necessary is this to merge in the PR? can make a task to fix this up in the future if needed

Reply:
nvm fixed it. It seems margin works slightly differently when used in sx than in plain css somehow.

Reply:
Thanks for looking into it! I'll note the margin thing in the future